CONTRIBUTING.md

```diff
@@ -1,16 +1,239 @@
-# CONTRIBUTE
-This is a short version of the [Development Procedures](https://github.com/RIOT-OS/RIOT/wiki/Development-procedures).
+# Contributing to RIOT
```

Comment:
As part of this PR, it would be useful to add a link to it in the CONTRIBUTE section of Readme.md

Reply:
Just a note (does not contradict your comment, but many don't notice): A link to this document always shows up under "Helpful resources" on the right whenever you open a new pull request. It's a pity GitHub doesn't make those links more prominent.

Reply:
Done